src/components/sort-radio-btns/SortRadioBtns.js

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from "react";
+import { ProductsContext } from "../../contexts/productsContext";
+import "./SortRadioBtns.css";
+
+function SortRadioBtns() {
+  const { sortBy, dispatch } = useContext(ProductsContext).products;
+  return (
+    <div className="radio-btn-wrapper">
+      <h5>Sort By Price: </h5>
+
+      <label className="radio-btn-container" for="low-to-high">
+        <input
+          type="radio"
+          id="low-to-high"
+          value="sort"
+          name="sort"
+          onChange={() => {
+            dispatch({ type: "SORT_BY", payload: "LOW_TO_HIGH" });
+          }}
```

Comment:
make a common function for both the sorts `high to low` and `low to high` and use that for common call